orchagent/mirrororch.cpp

```diff
@@ -368,7 +368,7 @@ bool MirrorOrch::getNeighborInfo(const string& name, MirrorEntry& session)
     // 3) Otherwise, return false.
     if (!m_neighOrch->getNeighborEntry(session.dstIp,
                 session.neighborInfo.neighbor, session.neighborInfo.mac) &&
-            (session.nexthopInfo.nexthop.isZero() ||
+            (session.nexthopInfo.nexthop.ip_address.isZero() ||
```

Comment:
As with acl redirect, mirror also would require a schema change to include interface. If it is for backward compatibility, then acl redirect doesn't seem to have backward compatibility

Reply:
Now mirror haven't support VRF, so have no schema change. Maybe mirror should add not interface but vrf in future, as the dest ip can be a remote host not neighbor and then to get nexthop from routes.